lunatrace/cli/fx/grypefx/store/gqlstorefx/map.go

```diff
@@ -93,24 +101,32 @@ introduced:
 	return ranges
 }
 
+func mapVersionConstraint(
+	affectedVersions []*gql.GetVulnerabilityVulnerabilityAffectedVulnerability_affectedAffected_versionsVulnerability_affected_version,
```

Comment:
holy christ these types lol. It might not be bad to alias these in another file as I have done for our TS generated types